specs/waku/waku-v2.md

```diff
@@ -190,10 +190,13 @@ message RPC {
 }
 
 message HistoryQuery {
-  // TODO Include time range, topic/contentTopic, limit, cursor, (request-id), etc
+  uint64 requestID = 1;
```

Comment:
Add explanation of this in the body below

Reply:
Why not use UUIDs?

Is this part in code already? (didn't see last time in PR)

Reply:
@oskarth yeah it is part of the code, I mean I only used a uint64 right now. Seems easy and flexible enough, we can change it in the future if feeling the need.

Reply:
The problem with uint64 is that it invites incrementing them or something, which would increase likelihood of collisions. Using UUIDs (or similar unique construct) seems pretty standard. Why not do it correct now?